.travis.yml

```diff
@@ -22,20 +20,20 @@ install:
   - go version
   - go env
   - go get -u github.com/matoous/godox
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin v1.17.1
   # Install Node 11 (required for WASM tests)
+  - wget https://raw.githubusercontent.com/creationix/nvm/v0.31.0/nvm.sh -O ~/.nvm/nvm.sh
   - source ~/.nvm/nvm.sh
   - nvm install 11
   - node --version
   - npm i -g yarn
   - yarn install
 
-before_install:
-  - wget https://raw.githubusercontent.com/creationix/nvm/v0.31.0/nvm.sh -O ~/.nvm/nvm.sh
-
-before_script:
-  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin v1.17.1
-
 script:
+  - bash .github/assert-contributors.sh
```

Comment:
@hugoArregui moved these to the top so they should fail faster! Hopefully cause less frustration with these :)

Reply:
hehe, thanks!